akka-docs/src/main/paradox/typed/actors.md

```diff
@@ -202,7 +202,7 @@ Scala
 Java
 :  @@snip [IntroSpec.scala](/akka-actor-typed-tests/src/test/java/jdocs/akka/typed/IntroTest.java) { #chatroom-protocol }
 
-Initially the client Actors only get access to an @scala[`ActorRef[GetSession]`]@java[`ActorRef<GetSession>`]
+Initially the client Actors only get access to an @apidoc[typed.ActorRef]@scala[`[GetSession]`]@java[`<GetSession>`]
```

Comment:
This can be written shorter as:

```suggestion
Initially the client Actors only get access to an @apidoc[typed.ActorRef[GetSession]]
```

... but this is OK also